embassy-stm32/src/rtc/mod.rs

```diff
@@ -0,0 +1,238 @@
+//! RTC peripheral abstraction
+use core::marker::PhantomData;
+
+#[cfg_attr(feature = "chrono", path = "datetime_chrono.rs")]
```

Comment:
It's strange that the API changes with or without chrono. We should try to make features additive. Could you change this so that we always use our own `DateTime`, and the `chrono` feature adds `From` impls to convert in both directions?

(Yes, many Cargo features we have are not additive, but it's still nice to make them additive when possible, and in this case it is)